docs/DeveloperGuide.md

```diff
@@ -471,7 +471,7 @@ _{Explain here how the data archiving feature will be implemented}_
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: 
+**Value proposition**:
 * To know when the guests check in and check out
 * To keep track of the guests' bills
 * To know where the guests book the hotel from
```

Comment:
Should we remove the "where guests book the hotel from" since it is not a feature in GuestBook?

Reply:
Agreed.